presto-jmx/src/main/java/io/prestosql/plugin/jmx/JmxTableHandle.java

```diff
@@ -105,6 +116,7 @@ public String toString()
                 .add("objectNames", objectNames)
```

Comment:
Found existing code has wrong combination in `toString()`.
```
.add("tableName", objectNames)
```

Reply:
Nice catch! I will create another pull request to fix it.

Reply:
Created: https://github.com/prestosql/presto/pull/634

Reply:
Thank you for your quick fix!